community/sgt-puzzles/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --with-gtk=3
+make
```

Comment:
blank line after ./configure ... and make; no blank line between make and make install